spec/controllers/families_controller_spec.rb

```diff
@@ -25,4 +25,14 @@
       end
     end
   end
+
+  describe "GET /families/:id" do
+    it "returns the show family" do
```

Comment:
Since there is a feature spec for it, I would remove this spec.

Reply:
👌